modules/woocommerce-analytics/wp-woocommerce-analytics.php

```diff
@@ -38,6 +39,14 @@ class Jetpack_WooCommerce_Analytics {
 	 * @return bool
 	 */
 	public static function shouldTrackStore() {
+		/**
+		 * Make sure WooCommerce is installed and active
+		 *
+		 * This action is documented in https://docs.woocommerce.com/document/create-a-plugin
+		 */
+		if ( !is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
```

Comment:
I would recommend that you keep using `Jetpack::get_active_plugins()` here; it accounts for more use-cases than `is_plugin_active` (e.g. multisite).

You can check the `get_active_plugins` function in `class.jetpack.php` to find out more.

Reply:
Thank you @jeherve I have checked it out and the change is reverted to Jetpack's get_active_plugins() function.